virocon/contours.py

```diff
@@ -739,9 +740,11 @@ class DirectSamplingContour(Contour):
 
     """
 
-    def __init__(self, model, alpha, n=100000, deg_step=5, sample=None):
+    def __init__(self, model, alpha, n='auto', deg_step=5, sample=None):
```

Comment:
I would prefer to set n=None as the default, meaning n is not set. 'auto' might imply that there are other possible string values.

Reply:
Okay, I was already considering whether to use None or 'auto' when writing the code.